async/kafka/group/group.go

```diff
@@ -122,6 +122,7 @@ func (c *consumer) Consume(ctx context.Context) (<-chan async.Message, <-chan er
 				closeConsumer(c.cg)
 				return
 			case consumerError := <-c.cg.Errors():
+				log.Warn("closing consumer group ", <-c.cg.Errors())
```

Comment:
Should we log `consumerError` instead of `<-c.cg.Errors()`. I'm not sure if we need the next (if there is a next) error from error channel.

Reply:
You're right actually, nice catch 👍  . Will proceed with the change